public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -55,10 +54,8 @@ export function UnifiedAlertList(props: PanelProps<UnifiedAlertListOptions>) {
     props.options.stateFilter.inactive = undefined; // now disable inactive
   }, [props.options.stateFilter]);
 
-  let dashboard: DashboardModel | undefined = undefined;
-
   useEffectOnce(() => {
-    dashboard = getDashboardSrv().getCurrent();
+    setDashboard(getDashboardSrv().getCurrent());
```

Comment:
without this new state , the dashboard value was lost after each render, and now we need to keep it for getting the time range from it.

Reply:
Could we use `onChangeTimeRange` to get the current `timeRange` instead of keeping the dashboard object?

Reply:
you're right 💯